### PR TITLE
Migrate camera doc to 4 0 0

### PR DESCRIPTION
**Pull request by kaltinril**
1. Added folders:
   1. /docs/features/Camera
   1. /docs/features/Camera/OthograhicCamera
1. Pulled in old documentation from Archived/Cameras
1. Split documentation into Camera/OrthograhicCamera pages.
   1. Created camera.md
   1. Created Camera/OrthographicCamera.md
1. Created 2 simple diagram/images
   1. If you don't want them I can remove them.
1. Added more information to the Camera page
1. Added more details and examples to the OrthographicCamera page:
   1. Creating an instance
   1. Moving the camera
   1. Zooming the camera
   1. Rotating the camera

Related Sample code pr: https://github.com/craftworkgames/MonoGame.Extended-samples/pull/14

### CODE REVIEW

File: docs/features/camera/camera.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+A camera has manage advantages, here are a few:
+1. Move the viewable area with an object (Like the player)
+1. Add a zoom effect (Perhaps entering or leaving a building)
+1. Add a rotation effect (Perhaps when entering a battle)
+1. Change to letterbox (Cinematic) view
+1. Create fluid motions with ease following a path
+
+## Further Reading
+
+[Matrix Basics](https://stevehazen.wordpress.com/2010/02/15/matrix-basics-how-to-step-away-from-storing-an-orientation-as-3-angles/)
```
Comment (AristurtleDev) [inline docs/features/camera/camera.md]: Make this a markdown list item 

```md
- list item
```

File: docs/features/camera/orthographic-camera/orthographic-camera.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+If you need to instead convert a world position into a screen position, you can use `WorldToScreen`.  Assuming you have a character in the world called `player`, you can use this.
+```csharp
+_screenPosition = _camera.WorldToScreen(new Vector2(player.X, player.Y));
+```
+
+
+## Further Reading
+
+[Matrix Basics](https://stevehazen.wordpress.com/2010/02/15/matrix-basics-how-to-step-away-from-storing-an-orientation-as-3-angles/)
+[Orthographic Camera](https://en.wikipedia.org/wiki/Orthographic_projection)
```
Comment (AristurtleDev) [inline docs/features/camera/orthographic-camera/orthographic-camera.md]: Change these to list items

Make this a markdown list item 

```md
- list item
- list item
```

**Review by AristurtleDev — CHANGES_REQUESTED** [review]
Two small changes requested. Regardless, good work, and thank you for the contribution.

File: docs/features/camera/camera.md (hunk 1 above)
Comment (kaltinril) [reply docs/features/camera/camera.md]: Man I didn't realize they were this small of changes, I've made them and pushed back up.

File: docs/features/camera/orthographic-camera/orthographic-camera.md (hunk 2 above)
Comment (kaltinril) [reply docs/features/camera/orthographic-camera/orthographic-camera.md]: I've pushed up the fixes here also.

File: docs/features/camera/camera.md (hunk 1 above)
Comment (AristurtleDev) [reply docs/features/camera/camera.md]: Yea, sorry it was super small changes

**Review by AristurtleDev — APPROVED** [review]

**Pull request merged**